### PR TITLE
feat(discover): add ability to query for avg decoded resource size and transfer size

### PR DESCRIPTION
**Pull request by DominikB2014**
You can now query for a resources
1. http.response_transfer_size
2. http.decoded_response_body_length

### CODE REVIEW

File: tests/snuba/api/endpoints/test_organization_events_stats_span_metrics.py [hunk 1]
```diff
@@ -216,6 +216,54 @@ def test_resource_size(self):
...
+                "yAxis": "avg(http.decoded_response_body_length)",
+                "project": self.project.id,
+                "dataset": "spansMetrics",
+                "excludeOther": 0,
+            },
+        )
+
+        data = response.data["data"]
+        assert response.status_code == 200
+        assert data == [(1697364000, [{"count": 0}]), (1697364060, [{"count": 4.0}])]
```
[the remark this answers is not in the document]
Comment (DominikB2014) [reply tests/snuba/api/endpoints/test_organization_events_stats_span_metrics.py:241]: Shoot, good call.
Was aware of this and fixed this issue before, just copied and pasted and old version of the test tbh. I'll update this

[the remark this answers is not in the document]
Comment (DominikB2014) [reply tests/snuba/api/endpoints/test_organization_events_stats_span_metrics.py]: @wmak updated, you can have a look now

File: src/sentry/testutils/cases.py [hunk 2]
```diff
@@ -1748,6 +1748,9 @@ class MetricsEnhancedPerformanceTestCase(BaseMetricsLayerTestCase, TestCase):
         "span.duration": "metrics_distributions",
         "span.self_time": "metrics_distributions",
         "http.response_content_length": "metrics_distributions",
+        "http.decoded_response_body_length": "metrics_distributions",
+        "http.decoded_response_body_length": "metrics_distributions",
```
Comment (wmak) [inline src/sentry/testutils/cases.py:1752]:
```suggestion
```
dupe

**Review by wmak — APPROVED** [review]

File: src/sentry/testutils/cases.py (hunk 2 above)
Comment (DominikB2014) [reply src/sentry/testutils/cases.py]: Thanks! removed the dup

**Pull request merged**